domains/atul.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Coming Soon",
+    "repo": "https://github.com/AtulPatil-me/Atulpatil",
+    "owner": {
+      "username": "Atul-Patil-99",
+      "email": "atuluttampatil.1757@gmail.com"
+    },
+    "record": {
+      "CNAME": "atulpatil-me.github.io/Atulpatil/"

```

Comment:
You cannot have slashes in the `CNAME`, If you want to use that URL try [URL redirection](https://github.com/is-a-dev/register/blob/main/docs/domains-json.md#record-required).

Reply:
Why is my PR not being approved by anyone else?